cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperator.java

```diff
@@ -1591,7 +1618,62 @@ public void checkCustomCaSecret(CertificateAuthority ca, Secret certSecret, Secr
         }
 
         Future<ReconciliationState> kafkaServiceEndpointReady() {
-            return withVoid(serviceOperations.endpointReadiness(namespace, kafkaService, 1_000, operationTimeoutMs));
+            Future<ReconciliationState> fut = withVoid(serviceOperations.endpointReadiness(namespace, kafkaService, 1_000, operationTimeoutMs));
```

Comment:
I'm not sure I understand why is this block needed.

Reply:
fixed